src/darwin/Framework/CHIP/MTROTAProviderDelegateBridge.h

```diff
@@ -33,6 +33,8 @@ class MTROTAProviderDelegateBridge : public chip::app::Clusters::OTAProviderDele
     // touches Matter objects.
     void Shutdown();
 
+    void ResetState();
```

Comment:
Again, this method is not actually implemented and should not be declared here.

Reply:
done